PlottingAnalysis/TS_Classify.m

```diff
@@ -281,6 +292,33 @@
     end
 end
 
+%-------------------------------------------------------------------------------
+% Save classifier:
+%-------------------------------------------------------------------------------
+if ~isempty(classifierFilename)
+  [Acc,Mdl,whatLoss] = GiveMeCfn(whatClassifier,TS_DataMat,...
+                            TimeSeries.Group,TS_DataMat,TimeSeries.Group,numClasses,[],[],true,0);
+                        
+  Operations = TS_GetFromData(whatData,'Operations');
+  jointClassifier.Operation.ID = Operations.ID;
+  jointClassifier.Operation.Name = Operations.Name;
+  jointClassifier.CVAccuracy = mean(foldLosses); % Cross-validated accuracy
+  jointClassifier.Accuracy = Acc; % For some reason the Acc retrieved as above is amazing?
```

Comment:
What do you mean by this? Should `Acc` be computed and stored in a more streamlined way with the bulk of this function?

Reply:
That comment should probably be removed. It was when I was training with around 1,000 samples, so because we had ~7000 features the accuracy was 100% (massive overfit).

I'm unsure what you mean re: streamlined storage?

Reply:
I guess I meant that the accuracy computed as `Acc` could maybe be computed as part of the body of this function…? (rather than only if saving the result to file)

Reply:
So Acc is the non-CV accuracy (for sake of completeness). So it is only really computed in the final stage here, if you want to save the classifier based on all training data

Reply:
Ok